### PR TITLE
TVAULT-5119 RHOSP datamover base image change

### PR DESCRIPTION
**Pull request by pushkar-mahajan**
@abhijeetpatra @shyam-biradar could you please review this

### CODE REVIEW

File: redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.2 [hunk 1]
```diff
@@ -16,10 +16,13 @@ USER root
...
 
 RUN dnf clean all
-RUN dnf config-manager --enable openstack-16-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms
-RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin -y
-RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
-RUN dnf config-manager --disable openstack-16-for-rhel-8-x86_64-rpms --disable rhel-8-for-x86_64-baseos-rpms
+RUN dnf config-manager --enable openstack-16.2-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms --enable advanced-virt-for-rhel-8-x86_64-rpms --enable rhceph-4-tools-for-rhel-8-x86_64-rpms --enable fast-datapath-for-rhel-8-x86_64-rpms
+RUN dnf install openstack-nova-common python3-nova python3-novaclient udev fuse -y
+RUN usermod -u 42436 nova && groupmod -g 42436 nova
+RUN usermod -aG disk nova && usermod -a -G libvirt nova && usermod -aG kolla nova
```
Comment (shyam-biradar) [inline redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.2:22]: @pushkar-mahajan You can use single command here.
usermod -a -G disk,libvirt,kolla nova

**Review by shyam-biradar — APPROVED** [review]
Changes looks good.

File: redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.2 [hunk 2]
```diff
@@ -18,10 +18,10 @@ ADD trilio.repo /etc/yum.repos.d/
 RUN dnf clean all
 RUN dnf config-manager --enable openstack-16.2-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms --enable advanced-virt-for-rhel-8-x86_64-rpms --enable rhceph-4-tools-for-rhel-8-x86_64-rpms --enable fast-datapath-for-rhel-8-x86_64-rpms
 RUN dnf install openstack-nova-common python3-nova python3-novaclient udev fuse -y
-RUN usermod -u 42436 nova && groupmod -g 42436 nova
-RUN usermod -aG disk nova && usermod -a -G libvirt nova && usermod -aG kolla nova
-RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin qemu-img ceph-common python3-libvirt-7.0.0-1.module+el8.4.0+9469+2eaf72bc python3-cinderclient -y
+RUN usermod -u 42436 nova && groupmod -g 42436 nova && usermod -aG disk,libvirt,kolla nova
+RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin qemu-img ceph-common python3-libvirt-7.0.0-1.module+el8.4.0+9469+2eaf72bc python3-cinderclient  qemu-kvm-block-rbd -y
 RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical --exclude=python3-libvirt
+RUN chown root:nova /usr/share/nova/nova-dist.conf
```
Comment (abhijeetpatra) [inline redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.2:24]: move this line to the place where we set all the permissions and ownership for other files

File: redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.1 [hunk 3]
```diff
@@ -16,10 +16,14 @@ USER root
...
-RUN dnf config-manager --enable openstack-16-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms
-RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin -y
-RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
-RUN dnf config-manager --disable openstack-16-for-rhel-8-x86_64-rpms --disable rhel-8-for-x86_64-baseos-rpms
+RUN dnf config-manager --enable openstack-16.1-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms --enable advanced-virt-for-rhel-8-x86_64-rpms --enable rhceph-4-tools-for-rhel-8-x86_64-rpms --enable fast-datapath-for-rhel-8-x86_64-rpms
+RUN dnf install openstack-nova-common python3-nova python3-novaclient udev fuse -y
+RUN usermod -u 42436 nova && groupmod -g 42436 nova && usermod -aG disk,libvirt,kolla nova
+RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin qemu-img ceph-common python3-libvirt-7.0.0-1.module+el8.4.0+9469+2eaf72bc python3-cinderclient  qemu-kvm-block-rbd -y
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical --exclude=python3-libvirt
+RUN chown root:nova /usr/share/nova/nova-dist.conf
```
Comment (abhijeetpatra) [inline redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.1:24]: move this line to the place where we set all the permissions and ownership for other files

**Review by abhijeetpatra — DISMISSED** [review]
why is the base image still same ?

**Comment by abhijeetpatra** [comment]
> why is the base image still same ?

my bad, I was checking on a specific commit and did not find your base changes there

**Review by abhijeetpatra — APPROVED** [review]

**Review by shyam-biradar — APPROVED** [review]
Looks good.

**Pull request merged**